recipes/gemini/meta.yaml

```diff
@@ -19,7 +20,7 @@ requirements:
     - {{ compiler('c') }}
   host:
     - pip
-    - python
+    - python >=3.8
```

Comment:
Updated because of ` nothing provides openssl >=1.0.2p,<1.0.3a needed by python-3.7.1-hd75c258_1001[0m`